File: app/src/main/java/com/bkjcb/rqapplication/emergency/adapter/EmergencyAdapter.java
package com.bkjcb.rqapplication.emergency.adapter;

import com.bkjcb.rqapplication.R;
import com.bkjcb.rqapplication.base.adapter.BaseRecycleViewAdapter;
import com.bkjcb.rqapplication.emergency.model.EmergencyModel;
import com.chad.library.adapter.base.BaseViewHolder;

/**
 * Created by DengShuai on 2020/7/8.
 * Description :
 */
public class EmergencyAdapter extends BaseRecycleViewAdapter<EmergencyModel, BaseViewHolder> {
    public EmergencyAdapter(int layoutResId) {
        super(layoutResId);
    }

    @Override
    protected void convert(BaseViewHolder helper, EmergencyModel item) {
        helper.setText(R.id.emergency_type,item.getReport_party())
                .setText(R.id.emergency_des,item.getMemo())
                .setText(R.id.emergency_address,item.getCompleteaddress())
                .setText(R.id.emergency_time,item.getReceivedate());
        if (item.getState().equals("已销根")){
            helper.setImageResource(R.id.emergency_type_img,R.drawable.vector_drawable_finished);
        }
    }
}

File: app/src/main/java/com/bkjcb/rqapplication/emergency/adapter/EmergencyAdapter.kt
package com.bkjcb.rqapplication.emergency.adapter

import android.text.TextUtils
import com.allen.library.SuperTextView
import com.bkjcb.rqapplication.R
import com.bkjcb.rqapplication.emergency.model.Emergency
import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder

/**
 * Created by DengShuai on 2017/8/28.
 */
class EmergencyAdapter(layoutResId: Int) : BaseQuickAdapter<Emergency, BaseViewHolder>(layoutResId) {
    override fun convert(holder: BaseViewHolder, emergency: Emergency) {
        val tv = holder.getView<SuperTextView>(R.id.third_content)
        tv.setLeftString(emergency.tel)
                .setRightString(emergency.unit)
        if (!TextUtils.isEmpty(emergency.area) && !TextUtils.isEmpty(emergency.remarks)) {
            tv.setCenterString(emergency.area + "(" + emergency.remarks + ")")
        } else if (!TextUtils.isEmpty(emergency.area) && TextUtils.isEmpty(emergency.remarks)) {
            tv.setCenterString(emergency.area)
        } else {
            tv.setCenterString("")
        }
    }
}